mmv1/products/networksecurity/FirewallEndpoint.yaml

```diff
@@ -0,0 +1,117 @@
+  guides:
+    'Firewall endpoint overview': 'https://cloud.google.com/firewall/docs/about-firewall-endpoints'
+    'Create and associate firewall endpoints': 'https://cloud.google.com/firewall/docs/configure-firewall-endpoints'
+timeouts: !ruby/object:Api::Timeouts
+  insert_minutes: 60
+  update_minutes: 60
+  delete_minutes: 60
+async: !ruby/object:Api::OpAsync
+  include_project: true
+  operation: !ruby/object:Api::OpAsync::Operation
+    base_url: '{{op_id}}'
+    result: !ruby/object:Api::OpAsync::Result
+      resource_inside_response: true
+autogen_async: true
+import_format: ['{{%parent}}/locations/{{location}}/firewallEndpoints/{{name}}']
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'network_security_firewall_endpoint_basic'
+    primary_resource_id: 'default'
+    skip_test: true   # update tests will take care of create and update. Firewall endpoint creation is too long.
+    min_version: beta
+    vars:
+      resource_name: 'my-firewall-endpoint'
+    test_env_vars:
+      org_id: :ORG_ID
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The name of the firewall endpoint resource.
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    required: true
+    immutable: true
+    description: |
+      The location (zone) of the firewall endpoint.
+    url_param_only: true
+  - !ruby/object:Api::Type::String
+    name: 'parent'
+    description: |
+      The name of the parent this firewall endpoint belongs to.
+      Format: organizations/{organization_id}.
+    immutable: true
+    url_param_only: true
+properties:
+  - !ruby/object:Api::Type::KeyValueLabels
+    name: 'labels'
+    description: |
+      A map of key/value label pairs to assign to the resource.
+  - !ruby/object:Api::Type::String
+    name: 'selfLink'
+    description: |
+      Server-defined URL of this resource.
+    output: true
+  - !ruby/object:Api::Type::Time
+    name: 'createTime'
+    description: Time the firewall endpoint was created in UTC.
+    output: true
+  - !ruby/object:Api::Type::Time
+    name: 'updateTime'
+    description: Time the firewall endpoint was updated in UTC.
+    output: true
+  - !ruby/object:Api::Type::Boolean
+    name: 'reconciling'
+    description: |
+      Whether reconciling is in progress, recommended per https://google.aip.dev/128.
+    output: true
+  - !ruby/object:Api::Type::Array
+    name: associatedNetworks
+    description: |
+      List of networks that are associated with this endpoint in the local zone.
+      This is a projection of the FirewallEndpointAssociations pointing at this
+      endpoint. A network will only appear in this list after traffic routing is
+      fully configured. Format: projects/{project}/global/networks/{name}.
+    item_type: Api::Type::String
+    output: true
+  - !ruby/object:Api::Type::Enum
+    name: 'state'
```

Comment:
Since this is output-only, it can be a string instead of an enum, which prevents us from needing to keep the set of possible values up-to-date.

Reply:
Makes sense. Updating it.